mayastor/src/subsys/mbus/mod.rs

```diff
@@ -31,10 +31,15 @@ pub fn mbus_endpoint(endpoint: Option<String>) -> Option<String> {
                 (endpoint.as_str(), 4222)
             };
 
+            debug!("Looking up nats endpoint {}...", address_or_ip);
             if let Ok(ipv4) = address_or_ip.parse::<Ipv4Addr>() {
-                lookup_addr(&IpAddr::V4(ipv4)).expect("Invalid Ipv4 Address");
+                let nats = lookup_addr(&IpAddr::V4(ipv4))
+                    .expect("Invalid Ipv4 Address");
+                debug!("Nats endpoint looked up at {}", nats);
```

Comment:
Could maybe word this as "Nats endpoint found at...."?

Reply:
Done